.circleci/config.yml

```diff
@@ -9,10 +9,16 @@ defaults: &defaults
     GOPATH: /go
```

Comment:
Similar to the vl3 case don't we want a step to build and push the kiknos based NSE?